util/err.c

```diff
@@ -5,10 +5,10 @@
 #include <errno.h>
 
 #include "err.h"
-#include "private.h"
+#include "stdlib.h"
```

Comment:
```suggestion

const char* libutil_getprogname (void);
```

The stdlib.h wrapper will not be included during the build. The forward declaration needs to be manually added here otherwise invalid results will occur due to C defaulting to assuming an int is returned, truncating the top half of the address.